.wordlist.txt

```diff
@@ -2,6 +2,7 @@ ABI
 activations
 AlexNet
 AMD
+amdclang
```

Comment:
See: https://github.com/RadeonOpenCompute/ROCm/pull/2078#issuecomment-1520459111

Reply:
Removed this word in https://github.com/RadeonOpenCompute/rocm-docs-core/pull/120/commits/7db8a5fbfad1beb6107a45138696961f09a0853a